pkg/snet/snettest/env.go

```diff
@@ -137,7 +137,7 @@ func createDmsgSrv(t *testing.T, dc disc.APIClient) (srv *dmsg.Server, srvErr <-
 	srv = dmsg.NewServer(pk, sk, dc)
 	errCh := make(chan error, 1)
 	go func() {
-		errCh <- srv.Serve(l, "")
+		errCh <- srv.Serve(l, "", 10)
```

Comment:
Is the signature of `srv.Serve` already changed somewhere? What do you think of extracting `10` to a constant with a describing name? I think it might improve the code readability if one can see what `10` means.

Reply:
May be resolved though